libs/blocks/quiz/quizcontainer.js

```diff
@@ -9,12 +9,12 @@ export const DecorateBlockBackgroundCmp = ({ background = '' }) => html`<div dat
       </picture>
   </div>`;
 
-export const DecorateBlockForeground = ({ heading, subhead }) => html`<div class="foreground container">
-    <div data-valign="middle" class="text" daa-lh="${heading}">
-      <h2 id="heading-xl-marquee-standard-medium-left" class="heading-XL">
+export const DecorateBlockForeground = ({ heading, subhead }) => html`<div class="quiz-foreground">
+    <div class="text" daa-lh="${heading}">
```

Comment:
I see this class name was there before, but `.text` sounds a bit generic, it might lead to issues when other blocks are present on the page

Reply:
No need for this class, or the wrapping div, for that matter. I've moved the daa-lh to the h3.